packages/worker/src/worker-tuner.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  FixedSizeSlotSupplier,
+  ResourceBasedTunerOptions,
+  WorkerTuner as NativeWorkerTuner,
+  SlotSupplier as NativeSlotSupplier,
+} from '@temporalio/core-bridge';
+import { Duration, msToNumber } from '@temporalio/common/lib/time';
+
+export type WorkerTuner = ResourceBasedTuner | TunerHolder;
+
+type ResourceBasedSlotsForType = ResourceBasedSlotOptions & {
+  tunerOptions: ResourceBasedTunerOptions;
```

Comment:
Haven't looked at TS, but is it common for us to reference types from the bridge project in user-facing code? Sure I know they are interfaces, but seems a bit off

Reply:
It's not that common but there's no meaningful issue. They would be exact dupes. If they ever need to diverge it can be done compatibly.

Reply:
> Haven't looked at TS, but is it common for us to reference types from the bridge project in user-facing types? Sure I know they are interfaces, but seems a bit off
AFAIK, only for `TelemetryOptions` and things under it.

> but there's no meaningful issue. [...] If they ever need to diverge it can be done compatibly.
At the very least, to avoid future problems, we must reexport those types from the worker package, and discourage users from ever importing them directly from the core package. But even then, that might still be a concern on the long term due to incoming links to the generated API docs, and therefore, we should generally try to avoid doing so. Now, in this specific case, given that the API is experimental, I don't mind just doing a reexport.